did_core/did_methods/did_resolver_sov/src/resolution/utils.rs

```diff
@@ -53,6 +53,21 @@ fn unix_to_datetime(posix_timestamp: i64) -> Option<DateTime<Utc>> {
         .map(|date_time| DateTime::<Utc>::from_naive_utc_and_offset(date_time, Utc))
 }
 
+fn expand_abbreviated_verkey(did: &str, verkey: &str) -> String {
+    if let Some(stripped_key) = verkey.strip_prefix('~') {
+        let decoded_did = bs58::decode(did).into_vec().unwrap();
+        let decoded_stripped_key = bs58::decode(stripped_key).into_vec().unwrap();
+        let decoded_did_string = String::from_utf8(decoded_did).unwrap();
+        let decoded_stripped_key_string = String::from_utf8(decoded_stripped_key).unwrap();
+
+        let decoded_verkey = format!("{}{}", decoded_did_string, decoded_stripped_key_string);
+
```

Comment:
not the biggest fan of treating raw bytes (Vec<u8> / &[u8]) as `String`s. I think you did this so that you could concatenate the bytes, which is correctly.

The `String::from_utf8` is error-prone here, since we aren't necessarily dealing with utf8 bytes, and this will panic if they aren't valid utf8 bytes.

there are other ways to concatenate bytes together (two vec<u8>'s )

Also in general; avoid `unwrap`s in non-testing code; you might have to make this return a `Result`